### ISSUE TITLE
Accessibility fixes

### ISSUE THREAD

**Issue by nathanrodman** [grafana/synthetic-monitoring-app#180]
* Probe locations need an `aria-label`

<img width="1034" alt="Screen Shot 2021-01-21 at 12 06 04 PM" src="https://user-images.githubusercontent.com/3469390/105406510-611d2000-5be1-11eb-905d-adba484cf19d.png">

---

* TCP use TLS settings needs `aria-label`
<img width="859" alt="Screen Shot 2021-01-21 at 12 11 17 PM" src="https://user-images.githubusercontent.com/3469390/105406834-d12ba600-5be1-11eb-810f-e27ffae1438c.png">

---

* Empty button for Query/Response. It should have some sort of text to inform the user what this button does.
<img width="859" alt="Screen Shot 2021-01-21 at 12 16 00 PM" src="https://user-images.githubusercontent.com/3469390/105407616-cf161700-5be2-11eb-9fa5-efc2662a591b.png">

---

**Comment by nathanrodman** [comment]
I looked into these issues further and they seem to all be issues within the core grafana ui components. I'm going to create a follow up issue in their project for this
